qchem/pennylane_qchem/qchem.py

```diff
@@ -173,12 +171,24 @@ def gen_meanfield_data(
 
 
 def gen_active_space(mol_name, hf_data, n_active_electrons=None, n_active_orbitals=None):
-    r"""Builds the active space for generating the Hamiltonian of the molecule.
+    r"""Builds the active space by partitioning the set of Hartree-Fock molecular orbitals.
 
     **Example usage:**
 
-    >>> gen_active_space('lih', './psi4/sto-3g', n_active_electrons=2, n_active_orbitals=2)
-    ([0], [1, 2])
+    >>> d_occ_orbitals, active_orbitals = gen_active_space('lih', './pyscf/sto-3g',
+    n_active_electrons=2, n_active_orbitals=2)
+    >>> print("List of doubly-occupied molecular orbitals: {:}".format(d_occ_indices))
+    >>> print("List of active molecular orbitals: {:}".format(active_indices))
+    >>> print("Number of qubits required to perform quantum simulations: {:}".format(2*len(
+    active_indices)))
+
+    List of doubly-occupied molecular orbitals: [0]
+    List of active molecular orbitals: [1, 2]
+    Number of qubits required to perform quantum simulations: 4
```

Comment:
With the python console formatting, each line following `>>>` directly displays the output (think of each `>>>` line as a jupyter notebook cell). This means the print statements aren't needed, and the order is slightly different:
```rest
>>> d_occ_orbitals, active_orbitals = gen_active_space('lih', './pyscf/sto-3g', n_active_electrons=2, n_active_orbitals=2)
>>> d_occ_indices # doubly-occupied molecular orbitals
[0]
>>> active_indices # active molecular orbitals
[1, 2]
>>> 2*len(active_indices) # number of qubits required for simulation
4
```

Reply:
Alternatively, if you want to display it as before, you could use two separate code blocks:

```rest
.. code-block:: python

    d_occ_orbitals, active_orbitals = gen_active_space('lih', './pyscf/sto-3g', ([0], [1, 2])
        n_active_electrons=2, n_active_orbitals=2)
    print("List of doubly-occupied molecular orbitals: {:}".format(d_occ_indices))
    print("List of active molecular orbitals: {:}".format(active_indices))
    print("Number of qubits required to perform quantum simulations: {:}".format(2*len(
    active_indices)))

Output:

.. code-block:: text

    List of doubly-occupied molecular orbitals: [0]
    List of active molecular orbitals: [1, 2]
    Number of qubits required to perform quantum simulations: 4
```
```